gitlab/v4/objects/merge_requests.py

```diff
@@ -388,6 +405,7 @@ class ProjectMergeRequestManager(CRUDMixin, RESTManager):
             "discussion_locked",
             "allow_maintainer_to_push",
             "squash",
+            "reviewer_ids",
```

Comment:
This list would also be nice to be updated.

https://docs.gitlab.com/ee/api/merge_requests.html#update-mr

The list of `update` attributes differs slightly from the list of `create` attributes.

Reply:
It would be really nice if GitLab published an OpenAPI spec, then this could probably be handled dynamically, or at least more easily 🤔

Reply:
I see you've found the upstream issue :)